.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish Ozone E2E Tests
+
+on:
+
+  workflow_dispatch:
+
+jobs:
+  publish-npm-release:
+    name: "publish to NPM"
+    runs-on: ubuntu-latest
+    if: github.repository == 'ozone-his/ozone-e2e'
+    permissions:
+      contents: read
+      id-token: write
```

Comment:
I don't think these are necessary. The `permissions` field is used to set the permissions for the `GITHUB_TOKEN` in a workflow run. In your case, it's granting read permissions for repository contents and write permissions for ID tokens. We aren't using `GITHUB_TOKEN` here, right?